frontend/src/data/config/MetricConfig.ts

```diff
@@ -38,30 +40,36 @@ export type MetricId =
   | "brfss_population_pct"
   | "copd_pct_share"
   | "copd_per_100k"
+  | "copd_ratio_age_adjusted"
   | "covid_cases"
```

Comment:
@benhammondmusic Do we need age adjusted metrics for each variable?

Reply:
Ya, the way everything is hooked together now the card looks for that property of metric id and I was getting a "something went wrong" error without them. I could look into getting the card to stop looking for that metric under certain conditions, but it seemed easier to just include the metric and have it be empty instead (since we are still displaying the card even if the metric isn't available)

Reply:
No worries, that makes sense.